acceptance/config/beaker/options.rb

```diff
@@ -8,6 +8,7 @@
   "puppetservice" => 'puppetserver',
   "use-service" => true,
   "master-start-curl-retries" => 60,
-  "puppetpath" => "/etc/puppetlabs/agent"
+  "puppetpath" => "/etc/puppetlabs/puppet",
+  "puppetserver-confdir" => '/etc/puppetlabs/puppetserver/conf.d'
```

Comment:
That's an unfortunate and interesting Rubyism.  A Ruby symbol cannot have a hyphen in it and one way to get around that is to convert the whole thing into a string, then symbolize the string.

I suspect it works with or without the colon because beaker just ends up calling `.to_s` on whatever it gets, so `:"puppetserver-confdir".to_s` and `'puppetserver-confdir'.to_s` have the same behavior in the end.

Personally I find `'puppetserver-confdir'` a little less confusing, and it works, so...